nanomq_cli/CMakeLists.txt

```diff
@@ -4,18 +4,31 @@ project(nanomq_cli)
 
 if(DEBUG AND ASAN)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fsanitize=address")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fsanitize=address")
 endif(DEBUG AND ASAN)
 
 if(DEBUG AND TSAN)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fsanitize=thread")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fsanitize=thread")
```

Comment:
add a condition here

Reply:
This only worked when debug & tsan option is ON，should we need add a condition?